docs/locale/ja/AdminTools.md

```diff
@@ -0,0 +1,2232 @@
+
+### どのように動作するか？
+
+前述の通り、Pulsar BrokerはPulsarのさまざまなエンティティを設定および管理するためのREST APIを公開しています。Pulsar admin CLIツールはadminコマンドを実行するために、Javaクライアントを使ってこれらのREST APIをコールします。利用可能なREST APIのリストについての詳細は*swaggerドキュメント*から学ぶことができます。
+
+## 利用準備
+
+### Pulsar admin CLIツール
+
+Pulsar Brokerが受け取ったリクエストに対して認証認可を行うことが可能である*Pulsarのセキュリティ*機能については他の章で述べました。adminツールはコマンドのリストを実行するためにPulsar BrokerのREST APIを呼びます。しかし、Pulsar Brokerのセキュリティが有効になっている場合、認証されたリクエストを得るためにBrokerのREST APIを呼ぶ際にadminツールは追加の情報を渡す必要があります。そこでこの情報が`conf/client.conf`ファイルにおいて正しく設定されていることを確認してください。
+
+これでPulsar admin CLIツールを使う準備が整いました。adminツールの利用を開始するため、下記のコマンドを実行してください。
+
+```shell
+$ bin/pulsar-admin --help
+```
+
+前章でPulsarのエンティティを管理するための他の代替手段 - REST APIとJava API - について述べました。このドキュメントではPulsarのエンティティを管理するためのREST APIのエンドポイントとJava APIのスニペットについても説明します。
+
+### REST API
+
+Pulsar admin REST APIの定義については*swaggerドキュメント*から学ぶことができます。このドキュメントでは各APIの利用方法とadmin CLIコマンドがどのようにREST APIに対応するかについて説明します。
+
+### Java API
+
+Java APIは```com.yahoo.pulsar.client.admin.PulsarAdmin```によりアクセスできます。
+
+下記のコードスニペットは*PulsarAdmin*の初期化の方法を示しています。後のドキュメントでPulsarのエンティティを管理するための使用方法について説明します。
+
+
+  ```java
+  URL url = new URL("http://localhost:8080");
+  String authPluginClassName = "com.org.MyAuthPluginClass"; //Pass auth-plugin class fully-qualified name if Pulsar-security enabled
+  String authParams = "param1=value1";//Pass auth-param if auth-plugin class requires it
+  boolean useTls = false;
+  boolean tlsAllowInsecureConnection = false;
+  String tlsTrustCertsFilePath = null;
+
+  ClientConfiguration config = new ClientConfiguration();
+  config.setAuthentication(authPluginClassName, authParams);
+  config.setUseTls(useTls);
+  config.setTlsAllowInsecureConnection(tlsAllowInsecureConnection);
+  config.setTlsTrustCertsFilePath(tlsTrustCertsFilePath);
+
+  PulsarAdmin admin = new PulsarAdmin(url, config);
+  ```
+
+### Pulsarエンティティ
+
+主にadminコマンドラインツールを使って下記のBrokerのエンティティにアクセスします。もしまだ下記のエンティティに対する理解が不十分なのであれば、[Pulsar入門](GettingStarted.md)で詳細を読むことができます。
+
+### Brokers
+
+アクティブなBrokerのリストおよび指定されたBrokerが所有するネームスペースのリストを取得できます。
+
+#### アクティブなBrokerリストの取得
+トラフィックを処理している利用可能でアクティブなBrokerを取得します。
+
+###### CLI
+
+```
+$ pulsar-admin brokers list use
+```
+
+```
+broker1.use.org.com:8080
+```
+
+###### REST
+
+```
+GET /admin/brokers/{cluster}
+```
+
+###### Java
+
+```java
+admin.brokers().getActiveBrokers(clusterName)
+```
+
+
+#### Brokerが所有するネームスペースリストの取得
+
+指定したBrokerが所有し処理しているすべてのネームスペースを取得します。
+
+###### CLI
+
+```
+$ pulsar-admin brokers namespaces --url broker1.use.org.com:8080 use
+```
+
+```json
+{
+    "my-property/use/my-ns/0x00000000_0xffffffff": {
+        "broker_assignment": "shared",
+        "is_controlled": false,
+        "is_active": true
+    }
+}
+```
+###### REST
+
+```
+GET /admin/brokers/{cluster}/{broker}/ownedNamespaces
+```
+
+###### Java
+
+```java
+admin.brokers().getOwnedNamespaces(cluster,brokerUrl)
+```
+
+
```

Comment:
recently we have added additional broker-api which we may want to add: [updateDynamicConfiguration](https://github.com/yahoo/pulsar/blob/master/docs/AdminTools.md#update-dynamic-configuration)